### PR TITLE
Added the steps for Running the site in Windows

### PR DESCRIPTION
**Pull request by chaitak-gorai**
This is new to me, so, please correct me if i make some mistakes. Thanks !

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -32,6 +38,13 @@ pip install -r requirements.txt
 ```
 python manage.py migrate
 ```
+if this gives error "Path not defined" remove the BASE DIR code from local_settings.py also rename database to like 'db_name'
```
Comment (justAcoderguy) [inline CONTRIBUTORS.md]: Could you please explain  why we are renaming the database? Thanks

Comment (chaitak-gorai) [reply CONTRIBUTORS.md]: Yes, it shouldn't be renamed. 

But can u suggest why it showing this error?
"  File "C:\Users\g12ch\Documents\GitHub\Covay\mysite\local_settings.py", line 3, in <module>
    BASE_DIR = Path(__file__).resolve().parent.parent
NameError: name 'Path' is not defined"

Comment (justAcoderguy) [reply CONTRIBUTORS.md]: Can you try with this once please and let me know :
```
import os

BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```
@chaitak-gorai Thanks

Comment (justAcoderguy) [reply CONTRIBUTORS.md]: If it works with this, then make a new PR with this code in the local_settings.py file and I'll merge it :)

**Pull request merged**